test/sql/attach/attach_filepath_roundtrip.test

```diff
@@ -67,14 +67,12 @@ DETACH con2_rollback_detach;
 statement error con1
 ATTACH '__TEST_DIR__/con2_rollback_detach.db';
 ----
-inferring the database type from the file header is not possible
+already attached
 
 # can't attach con1.db file in con2
 
 statement error con2
 ATTACH '__TEST_DIR__/con1.db';
-----
```

Comment:
Did you remove the error message here because different messages are possible now?

Reply:
Yes the output of this test has changed because the "already attached" error is no longer thrown because the database is not attached in that specific connection. I've added the expected error messages back. We might also need to change this test for Windows because of file locks but we'll see